Recon/src/Recon/Event/CompositePulse.cxx

```diff
@@ -2,37 +2,38 @@
 
 ClassImp(ldmx::CompositePulse)
```

Comment:
adding a semi-colon after this macro (even though its not required) will help avoid confusing clang format.
```suggestion
ClassImp(ldmx::CompositePulse);
```